src/patternfly/components/VerticalNav/styles.scss

```diff
@@ -0,0 +1,134 @@
+  --pf-c-vertical-nav__link--after--active--Height:  .25rem;
+  --pf-c-vertical-nav__link--after--active--Color:   var(--pf-global--Color--active);
+
+  // Vertical nav link colors
+  --pf-c-vertical-nav__link--Color:             var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--hover--Color:      var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--active--Color:     var(--pf-global--Color--active);
+  --pf-c-vertical-nav__link--focus--Color:      var(--pf-global--Color--dark-100);
+  --pf-c-vertical-nav__link--disabled--Color:   var(--pf-global--Color--disabled);
+
+  // Vertical nav link background colors
+  --pf-c-vertical-nav__link--BackgroundColor:           var(--pf-global--BackgroundColor--light-100);
+  --pf-c-vertical-nav__link--hover--BackgroundColor:    var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--active--BackgroundColor:   var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--focus--BackgroundColor:    var(--pf-global--BackgroundColor--light-200);
+  --pf-c-vertical-nav__link--disabled--BackgroundColor: var(--pf-global--BackgroundColor--disabled);
+
+  // Vertical nav link icon
+  --pf-c-vertical-nav__link__icon--MarginRight: var(--pf-global--spacer--xl);
+
+  // Vertical nav link arrow
+  --pf-c-vertical-nav__link__arrow--Top: calc(var(--pf-c-vertical-nav__link--PaddingTop) - .4rem);
+  --pf-c-vertical-nav__link__arrow--Right: var(--pf-global--spacer--xl);
+
+  /* stylelint-enable */
+
+  // List
+  color: var(--pf-c-vertical-nav--Color);
+  background-color: var(--pf-c-vertical-nav--BackgroundColor);
+
+  // List item
+  &__item {
+  }
+
+  // Link
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: baseline;
+    padding-top: var(--pf-c-vertical-nav__link--PaddingTop);
+    padding-right: var(--pf-c-vertical-nav__link--PaddingRight);
+    padding-bottom: var(--pf-c-vertical-nav__link--PaddingBottom);
+    padding-left: var(--pf-c-vertical-nav__link--PaddingLeft);
+    margin-top: var(--pf-c-vertical-nav__link--MarginTop);
+    margin-bottom: var(--pf-c-vertical-nav__link--MarginBottom);
+    color: var(--pf-c-vertical-nav__link--Color);
+    text-decoration: none;
+    background-color: var(--pf-c-vertical-nav__link--BackgroundColor);
+
+    // States
+    &:hover,
+    &:active,
+    &:focus {
+      text-decoration: none;
+    }
+
+    // Modifiers and states
+    &:hover,
+    &.pf-m-hover {
+      color: var(--pf-c-vertical-nav__link--hover--Color);
+      background-color: var(--pf-c-vertical-nav__link--hover--BackgroundColor);
+    }
+
+    &:active,
+    &.pf-m-active {
+      color: var(--pf-c-vertical-nav__link--active--Color);
+      background-color: var(--pf-c-vertical-nav__link--active--BackgroundColor);
+    }
+
+    &:focus,
+    &.pf-m-focus {
+      z-index: 2;
+      color: var(--pf-c-vertical-nav__link--focus--Color);
+      background-color: var(--pf-c-vertical-nav__link--focus--BackgroundColor);
```

Comment:
Should `:active` and `:focus` be reordered? Currently, if I place focus on the Home link or click on it, it turns from blue to dark gray (er, whatever that color is). Is that the expected behavior, or did we want it to stay blue (@kybaker)?

Reply:
I took a look and I agree with @jgiardino . I would expect the active state to take precedence so that the text "Home" would remain blue.